### PR TITLE
WIP WAG on moving to php 7.2.0 and openssl 1.1.0

### PR DESCRIPTION
**Pull request by craigcomstock**
ENT-3451

### CODE REVIEW

**Comment by nickanderson** [comment]
7.2 or 7.1.11?

**Comment by nickanderson** [comment]
i see 7.2 seems to have been released today!

File: deps-packaging/openssl/distfiles [hunk 1]
```diff
@@ -1 +1 @@
-f85123cd390e864dfbe517e7616e6566  openssl-1.0.2l.tar.gz
+27fb00641260f97eaa587eb2b80fab3647f6013b    openssl-1.1.0g.tar.gz
```
Comment (Lex-2008) [inline deps-packaging/openssl/distfiles]: it should be md5 checksums

**Comment by Lex-2008** [comment]
I think this can be closed now, given that all work is incorporated in  https://github.com/cfengine/buildscripts/pull/290

**Comment by craigcomstock** [comment]
this PR is deprecated.

**Pull request closed without merging**